stack/indexer/base/builder.sh

```diff
@@ -10,28 +10,43 @@
 
 set -e
 
-version="${1}"
-reference="${2}"
-BASE_DIR=/tmp/output/wazuh-indexer-base
+opensearch_version="${1}"
+future="${2}"
+revision="${3}"
+reference="${4}"
+base_dir=/tmp/output/wazuh-indexer-base
 
 # -----------------------------------------------------------------------------
 
+if [ -z "${revision}" ]; then
+    revision="1"
+fi
+
 # Including files
 if [ "${reference}" ];then
     curl -sL https://github.com/wazuh/wazuh-packages/tarball/"${reference}" | tar xz
     cp -r ./wazuh*/* /root/
+    version=$(curl -sL https://raw.githubusercontent.com/wazuh/wazuh-packages/${reference}/VERSION | cat)
+else
+    version=$(cat /root/VERSION)
+fi
+
+if [ "${future}" == "yes" ];then
```

Comment:
Same

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1648/commits/88b143bb8737d8b65fb27be9fe101805a0fe3d7a